doc/rm/c_cpp_coding_style.md

```diff
@@ -257,6 +257,36 @@ The code using an X Macro must:
     It is usually clearer if you split the definition over multiple lines.
 *   Immediately after the definition, include the `.def` file for the X Macro.
 
+### Undefined Behavior
+
+***All uses of Undefined Behavior (unless stated otherwise) are forbidden without explicit approval of toolchain experts.***
+
+As in any high-quality C/C++ project, Undefined Behavior (as defined by C++14 or C11 as the case may be) must be treated as a security risk.
```

Comment:
Let's not cast aspersions on other projects.

`Undefined behavior in a program (as defined by C++14 or C11) must be treated as a security risk. It also increases the likelihood of issues being encountered when changing toolchains. Undefined behavior must therefore be avoided unless absolutely necessary.`